src/test/codegen/vtabletype.rs

```diff
@@ -14,7 +14,7 @@
 // ignore-tidy-linelength
 // ignore-windows
 // ignore-macos
-// min-system-llvm-version 5.1
+// min-llvm-version 6.0
```

Comment:
Oops, I don't know if I messed this up. Like LLVM 5.1 does not exist, are we sure whatever change was needed was not part of LLVM 6.0 and that the patch will be backported to LLVM 5.1 if it is ever released (which seems unlikely) ?

Reply:
That was part of #45897, which included rust-lang/llvm#97 with [D39503](https://reviews.llvm.org/D39503).  That patch is indeed included in LLVM 6.0, and I don't think there's any point tagging this for a hypothetical 5.1.